src/covid_updater/scraping/countries/germany.py

```diff
@@ -15,16 +15,16 @@ def __init__(self):
             },
             column_renaming={
                 "state": "region",
-                "initialDosesCumulative": "people_vaccinated",
-                "finalDosesCumulative": "people_fully_vaccinated",
+                "atLeastPartiallyVaccinatedCumulative": "people_vaccinated",
```

Comment:
I'm not sure which metric you want exactly: is it `atLeastPartiallyVaccinatedCumulative` or `onlyPartiallyVaccinatedCumulative`? See https://github.com/mathiasbynens/covid-19-vaccinations-germany#partially-vs-fully-vaccinated. Let me know and I'd be happy to update the patch.

Reply:
Hi, and - once again - thanks @mathiasbynens!

I am currently reporting three metrics (same as OWID):
- `total_vaccinations` := Cumulative number of doses administered.
- `people_vaccinated` := Cumulative number of people that received >= 1 dose (either first one in 2-dose-scheme doses or one and only in 1-dose-scheme).
- `people_fully_vaccinated` := Cumulative number of people that are fully vaccinated (received 2 doses in 2-dose-scheme or 1 dose in 1-dose scheme).

Having a look at your definitions, I'd say that:

```
total_vaccinations = onlyPartiallyVaccinatedCumulative + fullyVaccinatedCumulative
people_vaccinated = atLeastPartiallyVaccinatedCumulative
people_fully_vaccianted = fullyVaccinatedCumulative
```

Therefore some small modifications in the patch would be required. I'll point them as comments